### ISSUE TITLE
Voting out "schism-creators" in the GOP Central Committee

### ISSUE THREAD

**Issue by rickewest** [utahCountyP45/utahCountyP45.github.io#7]
One of the main responsibilities of our precinct delegates is that they get the opportunity to vote for members of the Utah Republican Central Committee. I am gravely concerned about a small group of this committee that is creating a deep schism within the party. They represent the far right of the party, but have shown blatant disregard for the opinions of the rest of us in the party and plundered our party into debt just so they can thumb their nose at the will of the people on the Count My Vote issue. They are the "Group of 51" that called special meetings at the last minute that others could not attend so they could push through their radical positions. I believe their actions are very contradictory to the democratic process. Whether you agree with count my vote or not, secret meetings and expensive lawsuits are not how to run a party. I hope our delegates will inform themselves on who these representatives are, and vote them off the Central Committee and bring sanity back to the leadership of the Utah GOP.

**Comment by jjamesstark** [comment]
I am very concerned with any institution or process that undermines our democracy (in any sense) or centralizes power away from our communities. 

I am a novice when it comes to the GOP, but this isn't fmthe first time I've heard of the issue and I have the same attitude as you.

What I'd like to do, with your help and the help of others, is organize before both conventions to identify all threats to our democracy and together make voting resolution. That is, rules and principles our precinct expects all delegates to keep while voting at convention.

How does that sound?

**Comment by rickewest** [comment]
yep! My understanding is that you can't vote for new members of the central committee for 1.5 years. But what you can do is support Diane Christensen's proposal to limit the power of the Group of 51 ... basically changing the by-laws to require a greater majority be present for major policies. She could use your support, so ask her about her efforts there. Then, in 1.5 years, help get better people on that Central Committee!

Another important thing to do may be the work with other delegates, mostly with the other state delegates, as my understanding is most of the Group of 51 is from outside Provo/Orem, so we need to help inform the delegates from other areas about what is going on so they aren't asleep at the wheel when these folks go up for re-election. I can send you the list of who it is if you want it (the ones organizing these secret meetings). Here locally, Chris Herrod has been a ring-leader for them.
    On Sunday, April 1, 2018, 3:41:23 PM MDT, jstark <notifications@github.com> wrote:  
 
 
I am very concerned with any institution or process that undermines our democracy (in any sense) or centralizes power away from our communities.

I am a novice when it comes to the GOP, but this isn't fmthe first time I've heard of the issue and I have the same attitude as you.

What I'd like to do, with your help and the help of others, is organize before both conventions to identify all threats to our democracy and together make voting resolution. That is, rules and principles our precinct expects all delegates to keep while voting at convention.

How does that sound?

**Comment by jjamesstark** [comment]
https://github.com/utahCountyP45/utahCountyP45.github.io/blob/master/js_notes/gop_central_comm.md

See notes
